ja/refresher-linear-algebra.md

```diff
@@ -0,0 +1,342 @@
+<br>
+
+**15. inner product: for x,y∈Rn, we have:**
+
+&#10230;
+内積: x、y∈Rnについては、
+<br>
+
+**16. outer product: for x∈Rm,y∈Rn, we have:**
+
+&#10230;
+外積: x∈Rm,y∈Rnについては、
+<br>
+
+**17. Matrix-vector ― The product of matrix A∈Rm×n and vector x∈Rn is a vector of size Rn, such that:**
+
+&#10230;
+行列-ベクトル - 行列A∈Rm×nとベクトルx∈Rnの積はサイズRnのベクトルで、次のようになります。
+<br>
+
+**18. where aTr,i are the vector rows and ac,j are the vector columns of A, and xi are the entries of x.**
+
+&#10230;
+ここで、aTr、iはAのベクトル行、ac、jはAのベクトル列です。 xiはxのエントリです。
+<br>
+
+**19. Matrix-matrix ― The product of matrices A∈Rm×n and B∈Rn×p is a matrix of size Rn×p, such that:**
+
+&#10230;
+行列-行列 - 行列A∈Rm×nとB∈Rn×pの積は次のようにサイズRm×pの行列です。 (There is a typo in the original: Rn×p)
+<br>
+
+**20. where aTr,i,bTr,i are the vector rows and ac,j,bc,j are the vector columns of A and B respectively**
+
+&#10230;
+aTr、i、bTr、iはベクトル行。　ac、j、bc、jはそれぞれAとBのベクトル列です。
+<br>
+
+**21. Other operations**
+
+&#10230;
+その他の演算
+<br>
+
+**22. Transpose ― The transpose of a matrix A∈Rm×n, noted AT, is such that its entries are flipped:**
+
+&#10230;
+転置 ― A∈Rm×nの転置行列はATと示される。　Aの行列要素が交換されます。
+<br>
+
+**23. Remark: for matrices A,B, we have (AB)T=BTAT**
+
+&#10230;
+備考： 行列AとBの場合、（AB）T = BTAT** となります。
+<br>
+
+**24. Inverse ― The inverse of an invertible square matrix A is noted A−1 and is the only matrix such that:**
+
+&#10230;
+逆行列 ― 可逆正方行列Ａの逆行列はＡ − １と表される。 以下を満たす唯一の行列です。
+<br>
+
+**25. Remark: not all square matrices are invertible. Also, for matrices A,B, we have (AB)−1=B−1A−1**
+
+&#10230;
+備考： すべての正方行列が可逆的なわけではありません。　行列A、Bについては、(AB)−1=B−1A−1
+<br>
+
+**26. Trace ― The trace of a square matrix A, noted tr(A), is the sum of its diagonal entries:**
+
+&#10230;
+跡 ― 正方行列Aの跡は、その対角要素の合計です。　tr(A)と表される。
+<br>
+
+**27. Remark: for matrices A,B, we have tr(AT)=tr(A) and tr(AB)=tr(BA)**
+
+&#10230;
+備考： 行列A、Bの場合：　tr(AT)=tr(A)とtr(AB)=tr(BA)となります。
+<br>
+
+**28. Determinant ― The determinant of a square matrix A∈Rn×n, noted |A| or det(A) is expressed recursively in terms of A∖i,∖j, which is the matrix A without its ith row and jth column, as follows:**
+
+&#10230;
+行列式 ― 行列式は|A| または det(A) と表される。 正方行列　A∈Rn×n　の行列式はAijによって再帰的に表現されます。
+ それはi番目の行とj番目の列のない行列Aです。 次のように：
+<br>
+
+**29. Remark: A is invertible if and only if |A|≠0. Also, |AB|=|A||B| and |AT|=|A|.**
+
+&#10230;
+備考： |A|≠0の場合に限り、行列は可逆行列です。また|AB|=|A||B| と |AT|=|A|。
+<br>
+
+**30. Matrix properties**
+
+&#10230;
+行列の性質
+<br>
+
+**31. Definitions**
+
+&#10230;
+定義
+<br>
+
+**32. Symmetric decomposition ― A given matrix A can be expressed in terms of its symmetric and antisymmetric parts as follows:**
+
+&#10230;
+対称分解 ― 行列Aは次のように対称および反対称部分で表現できます。
+<br>
+
+**33. [Symmetric, Antisymmetric]**
+
+&#10230;
+[対称、反対称]
+<br>
+
+**34. Norm ― A norm is a function N:V⟶[0,+∞[ where V is a vector space, and such that for all x,y∈V, we have:**
+
+&#10230;
+ノルムは関数N:V⟶[0,+∞[　Vはベクトル空間、　すべてのx、y∈Vについて：
+]]
+<br>
+
+**35. N(ax)=|a|N(x) for a scalar**
+
+&#10230;
+N(ax)=|a|N(x) スカラー用
```

Comment:
Looking at the pdf, it appears that "a" in "for a scalar" refers to the variable a.
```suggestion
スカラー a に対して N(ax)=|a|N(x) 
```